_episodes/02-installation.md

```diff
@@ -1,119 +1,192 @@
+First, open a bash terminal and create a directory to install Julia in
 
-If everything was installed properly, ESMValTool should have printed a
-help message to the console.
+```bash
+mkdir ~/julia
+```
 
-For a more complete installation verification, run the automated tests and
-confirm that no errors are reported:
+Next, download the file
+[`julia-1.0.5-linux-x86_64.tar.gz`](https://julialang-s3.julialang.org/bin/linux/x64/1.0/julia-1.0.5-linux-x86_64.tar.gz)
+by clicking the link or going to the [Julia downloads page](https://julialang.org/downloads/).
+To extract the file, you can use the following command:
 
-~~~
-    python setup.py test --installation
-~~~
-{: .source}
+```bash
+tar -xvzf ~/Downloads/julia-1.0.5-linux-x86\_64.tar.gz -C ~/julia
+```
 
-{% include links.md %}
+This will extract the files to a folder named `~/julia/julia-1.0.5`.
+To run Julia, you need to add the full path of Julia's `bin` folder to PATH environment variable.
+To do this, you can edit the `~/.bashrc` (or `~/.bash_profile`) file.
+Open the file in your favorite editor and add a new line as follows at the bottom of the file:
+
+```bash
+export PATH="$PATH:$HOME/julia/julia-1.0.5/bin"
+```
+
+Finally, for the settings to take effect, either reload your bash profile
+
+```bash
+source ~/.bashrc
+```
+
+(or `source ~/.bash_profile`), or close the bash terminal window and open a new one.
+
+To check that the Julia executable can be found, run
+
+```bash
+which julia
+```
+
+to display the path to the Julia executable, it should be `~/julia/julia-1.0.5/bin/julia`.
+To test that Julia is installed correctly, run
 
+```bash
+julia
+```
+
+to start the interactive Julia interpreter. Press `Ctrl+D` to exit.
+
+## Install the ESMValTool package
+
+To install the ESMValTool package, run
+
+```bash
+conda create -n esmvaltool -c conda-forge -c esmvalgroup esmvaltool
+```
+
+This will create a new
+[Conda environment](https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html)
+called `esmvaltool`, with the ESMValTool package and all of its dependencies installed in it.
+
+> ## Conda common problems
+>
+> Below are some common problems that could happen while installing.
+>
+> -   installation takes long time,
+>     -   downloads and compilations can take several minutes to complete, please be patient
+>     -   you might have bad luck and the dependencies can not be resolved at the moment, please try again later or [raise an issue](https://github.com/ESMValGroup/ESMValTool/issues/new/choose)
+> -   if you have a old conda installation you could get a `UnsatisfiableError` error. Please install a new conda and try again
+> -   downloads fail due to company proxy, see [conda docs](https://docs.anaconda.com/anaconda/user-guide/tasks/proxy/) how to resolve
+>
+{: .callout}
+
+## Test that the installation was successful
+
+To test that the installation was successful, run
+
+```bash
+conda activate esmvaltool
```

Comment:
Again, some example output code blocks might be nice

Reply:
Activation changes the terminal prompt, I think there are to many different prompt so I explained the change instead of showing a prompt.